src/VisualStudio/Core/Def/RoslynPackage.cs

```diff
@@ -194,7 +195,9 @@ private async Task LoadOptionPersistersAsync(IComponentModel componentModel, Can
 
             foreach (var provider in persisterProviders)
             {
-                _ = await provider.GetOrCreatePersisterAsync(cancellationToken).ConfigureAwait(true);
+                var persister = await provider.GetOrCreatePersisterAsync(cancellationToken).ConfigureAwait(true);
+                if (persister is PackageSettingsPersister packageSettingsPersister)
+                    packageSettingsPersister.Initialize(this);
```

Comment:
consider doc'ing in some form.  like why we're special casing this here.  not a huge deal.